src/applications/gi-sandbox/containers/ComparePage.jsx

```diff
@@ -525,6 +525,28 @@ export function ComparePage({
             facilityCodes={loaded}
             showDifferences={showDifferences}
             fieldData={[
+              {
+                label: 'Length of VET TEC programs',
+                mapper: institution => {
+                  if (
+                    institution.programLengthInHours &&
+                    institution.programLengthInHours.length > 0
+                  ) {
+                    const maxHours = Math.max(
+                      ...institution.programLengthInHours,
+                    );
+                    const minHours = Math.min(
+                      ...institution.programLengthInHours,
+                    );
+                    return `${
+                      minHours === maxHours
+                        ? minHours
+                        : `${minHours} - ${maxHours}`
+                    } hours`;
+                  }
+                  return 'N/A';
+                },
```

Comment:
Can we move this to a constant to make things cleaner?

Reply:
Sure